### PR TITLE
DM-35896: Use gen3 butler in examples

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: doc/lsst.display.firefly/viewing-footprints.rst [hunk 1]
```diff
@@ -13,9 +13,9 @@ Define a Butler instance and an identifier for retrieving the data.
 
 .. code-block:: py
 
-    from lsst.daf.persistence import Butler
+    from lsst.daf.butler import Butler
     butler = Butler('/datasets/hsc/repo/rerun/RC/w_2019_22/DM-19244/')
```
Comment (arunkannawadi) [inline doc/lsst.display.firefly/viewing-footprints.rst:17]: This is likely to be outdated and should be changed as well.

File: doc/lsst.display.firefly/getting-started-lsp.rst [hunk 2]
```diff
@@ -34,9 +34,9 @@ Retrieve a simulated LSST image.
 .. code-block:: py
     :name: getting-started-getimage
 
-    from lsst.daf.persistence import Butler
+    from lsst.daf.butler import Butler
     butler = Butler('/project/shared/data/Twinkles_subset/output_data_v2')
```
Comment (arunkannawadi) [inline doc/lsst.display.firefly/getting-started-lsp.rst:38]: I'm not sure if this is/was a Gen3 butler, but defining a REPO and a COLLECTION would be more illustrative.

**Review by arunkannawadi — APPROVED** [review]

File: doc/lsst.display.firefly/getting-started-lsp.rst (hunk 2 above)
Comment (timj) [reply doc/lsst.display.firefly/getting-started-lsp.rst]: I changed it to use collection and also realized that the butler.get needed to be fixed.

Comment (arunkannawadi) [reply doc/lsst.display.firefly/getting-started-lsp.rst]: I missed the `butler.get` previously. Good catch.

**Pull request merged**